cypress/e2e/tests/pages/global-settings/settings-p2.spec.ts

```diff
@@ -224,9 +224,10 @@ describe('Settings', { testIsolation: 'off' }, () => {
     settingsPage.settingsValue('ui-offline-preferred').contains(settings['ui-offline-preferred'].original);
   });
 
-  it.skip('[Vue3 Skip]: can update ui-brand', { tags: ['@globalSettings', '@adminUser'] }, () => {
-    const rancherLogo = '/img/rancher-logo.66cf5910.svg';
-    const suseRancherLogo = '/img/rancher-logo.055089a3.svg';
+  it('can update ui-brand', { tags: ['@globalSettings', '@adminUser'] }, () => {
+    // We probably want a better way to distinguish between rancher and suse logos. I'm doing this as part of the vue3 migration and trying to keep things as similar as possible.
```

Comment:
The hashes were changing between both my localhost instance and the cypress tests. It didn't seem like we could rely on the hashes anymore.

Reply:
Yeah, I could definitely see how the vue3 migration might be a scenario that could cause the hash to change. It's probably best that we don't rely on the hashes for the test.